opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/Ctags.java

```diff
@@ -248,22 +248,26 @@ private void addRustSupport(List<String> command) {
         }
 
         // The following are not supported yet in Universal Ctags b13cb551
+        command.add("--kinddef-rust=C,const,Static\\ constants");
+        command.add("--kinddef-rust=I,impl,Trait\\ implementation");
+        command.add("--kinddef-rust=r,trait,Traits");
+        command.add("--kinddef-rust=V,variable,Variables");
```

Comment:
I'm seeing another warning:
```
Error from ctags: ctags: Warning: the kind for name `variable' specified in "--kinddef-rust" option is already defined.
```

Maybe we will have to pick another name or maybe just skip the kind definition?

Reply:
I must have missed this one. Looking at the existing definitions I get:

```
Rust                 M      macro                 yes     no      0      NONE   Macro Definition
Rust                 P      method                yes     no      0      NONE   A method
Rust                 c      implementation        yes     no      0      NONE   implementation
Rust                 e      enumerator            yes     no      0      NONE   An enum variant
Rust                 f      function              yes     no      0      NONE   Function
Rust                 g      enum                  yes     no      0      NONE   Enum
Rust                 i      interface             yes     no      0      NONE   trait interface
Rust                 m      field                 yes     no      0      NONE   A struct field
Rust                 n      module                yes     no      0      NONE   module
Rust                 s      struct                yes     no      0      NONE   structural type
Rust                 t      typedef               yes     no      0      NONE   Type Alias
Rust                 v      variable              yes     no      0      NONE   Global variable
```
You see there is already a `v` defined, but I'm not sure if the newly defined `V` is another type of variable? in this case we should just use another shortname in the kinddef? Otherwise the whole kinddef can be dropped and we use the the regex options `v` instead of `V`.

Reply:
I've tested with this snippet found here: https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html
```rust
fn main() {
    let x = 5;
    println!("The value of x is: {}", x);
    x = 6;
    println!("The value of x is: {}", x);
}
```
and ctags did not recognize the variable `x` by default. Funny thing is that OpenGrok has a bug in the regex as well as it should probably be:
`/^[[:space:]]*let([[:space:]]+mut)?[[:space:]]+([[:alnum:]_]+)/\\2/V/`

Either way, I'd personally go with a new name like `localVariable`. What do you think?

Reply:
Makes sense. Sorry didn't get around to try this out myself. I applied the fix.